### PR TITLE
Enable close to tray on win*

### LINKED ISSUE #169: Minimize to tray

### PR DESCRIPTION
**Pull request by GrayHatter**
closes #169

### CODE REVIEW

File: util.c [hunk 1]
```diff
@@ -565,6 +565,7 @@ typedef struct
     uint16_t proxy_port;
     uint8_t proxyenable;
     uint8_t logging_enabled : 1;
+    uint8_t close_to_tray;
```
Comment (irungentoo) [inline util.c]: This save struct was put there when I increased the size of the utox settings save format but didn't want people to loose their settings. It's for an old save version.

File: main.h [hunk 2]
```diff
@@ -42,6 +42,7 @@ typedef struct
     uint16_t proxy_port;
     uint8_t proxyenable;
     uint8_t logging_enabled : 1;
+    uint8_t close_to_tray;
```
Comment (irungentoo) [inline main.h]: Use a bit from one of the 4 zero ones below to not break the utox save. Those are bitfields by the way.

**Comment by fcore117** [comment]
this is news of century, thanks developers. Next linux...

**Comment by Enelar** [comment]
@fcore117 indeed, new era

**Pull request merged**